apps/web/tailwind.config.ts

```diff
@@ -4,7 +4,7 @@ import type { Config } from "tailwindcss";
 import sharedConfig from "@repo/tailwind-config";
 
 const config: Pick<Config, "content" | "presets"> = {
-  content: ["./app/**/*.tsx"],
+  content: ["./app/**/*.tsx", "../../packages/ui/**/*{.js,.ts,.jsx,.tsx}"],
```

Comment:
Fixes an issue where tailwind occasionally wouldn't work in `web`.
https://www.willliu.com/blog/Why-your-Tailwind-styles-aren-t-working-in-your-Turborepo